scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.html

```diff
@@ -33,12 +33,12 @@
 
         <div [ngClass]="isPanelCollapsed ? 'sc-space-top col-sm-12' : 'sc-space-top col-sm-8'">
             <p>
-                <span *ngIf="(usecases|filter: searchTerm).length === usecases.length">{{usecases.length}}</span>
-                <span *ngIf="(usecases|filter: searchTerm).length !== usecases.length">Found {{(usecases|filter: searchTerm).length}} of {{usecases.length}}</span>
+                <span *ngIf="(usecases|scFilterArray: searchTerm).length === usecases.length">{{usecases.length}}</span>
```

Comment:
this looks duplicated with the scenrio view - maybe something to be extracted into a component but I'm not sure if it's really that similar

Reply:
You're right. I will start a new branch to divide the table into several components as soon as I'm finished with the detailarea...